core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -615,81 +653,75 @@ var ExplorationEditorMainTab = function() {
     }
   };
 
+  /**
+   * Parse for Angular selectors and remove them.
```

Comment:
Are you removing or replacing them with "..."?

Reply:
It will either be '...' or the text input by user 
`contains "hello world"` 
This is for displaying the interaction rule.
or
`contains ...`
This is for displaying the Add Response's rule descriptions (the drop down elements)

Reply:
OK, it sounds like "replace", but your docstring says "remove". Maybe add more info to make it clearer?